CRM/Donrec/Exporters/EmailPDF.php

```diff
@@ -140,6 +140,21 @@ protected function sendEmail($receipt, $pdf_file, $is_test, $snapshot_line_id) {
       // load contact data
       $contact = civicrm_api3('Contact', 'getsingle', array('id' => $receipt['contact_id']));
 
+      if (CRM_Donrec_Logic_Settings::get('email_location_type_id')) {
+        // load email address from the configured location type
+        $email_location_type_id = CRM_Donrec_Logic_Settings::get('email_location_type_id');
+        $email = \Civi\Api4\Email::get(FALSE)
+          ->addSelect('email')
+          ->addWhere('location_type_id', '=', $email_location_type_id)
+          ->addWhere('contact_id', '=', $contact['id'])
+          ->setLimit(1)
```

Comment:
Instead of setting a limit (when reducing the result with `first()` anyway) I'd add sorting to make the result more predictable in case of multiple results.